src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -158,21 +253,6 @@ class VisitService(
     return VisitDto(visitEntity)
   }
 
-  fun deleteVisit(reference: String) {
-    val visit = visitRepository.findByReference(reference)
-
-    visit?.let {
-      visitRepository.delete(it)
-      telemetryClient.trackEvent(
-        "visit-scheduler-prison-visit-deleted",
-        mapOf(
-          "reference" to reference
-        ),
-        null
-      )
-    } ?: run { log.debug("Visit reference $reference not found") }
-  }
-
   fun deleteAllVisits(visits: List<VisitDto>) {
```

Comment:
This method needs to take the application reference instead of reference when called from the delete task?

Reply:
There is another task for this not part of this review